book/website/welcome.md

```diff
@@ -1,13 +1,20 @@
 (welcome)=
 # Welcome
 
-_The Turing Way_ is a handbook to reproducible, ethical and collaborative data science. 
+*Welcome to The Turing Way guides to reproducible, ethical and collaborative data science.*
 
-The project is open source (anyone can [inspect the code](https://github.com/alan-turing-institute/the-turing-way)), open contribution (anyone can {ref}`contribute<ch-contributing>`), and open licenced (anyone can [re-use the materials](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
+_The Turing Way_ is an open source, open collaboration and community-driven project.
+We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
+Our goal is to provide all the information that researchers and data scientists in academia, industry and the public sector need at the start of their projects to ensure that they are easy to reproduce at the end.
 
-Our goal is to provide all the information that data scientists in academia, industry, government and the third sector need to ensure that the projects they work on are easy to reproduce and reuse.
+```{note}
+_The Turing Way_ is not meant to be read from start to finish. 
+Start with a concept, tool or method that you need now, in your current work. 
```

Comment:
Shame to lose the buffet reference, but I really like this new wording ❤️

Reply:
Thanks, Danny. I like the buffet reference too, but I want to avoid jargon in open research generated within Europe and the USA context, that may not make a lot of sense for global readers. I know it is too picky, but I hope this rephrasing doesn't lose the meaning.

Reply:
It keeps the meaning beautifully. And you're right - less jargon / 'in jokes' the better! I hadn't thought about it but it's possible as well that the metaphor itself might only work in certain cultures.

Reply:
Thank you for being patient with this PR :)

Reply:
It feels important to get the landing page right - first impressions count! :)
Very happy to be able to help